src/legacy/core_plugins/kibana/public/discover/np_ready/_discover.scss

```
@@ -141,6 +141,7 @@ discover-app {
 .dscSidebarItem__label {
   overflow: hidden; /* 1 */
   text-overflow: ellipsis; /* 1 */
+  width: 100%;
```

👍 But if you want something that's more programattic, EUI has a mixin called `euiTextTruncate` for this case so you **could** replace all these with:
```suggestion
  @include euiTextTruncate;
```

---

thx, decided to merge it as it is, since it would need another test round, even if I'm sure it worked this way